packages/mds-cache/index.ts

```diff
@@ -256,33 +256,19 @@ async function hreads(
   // bleah
   const multi = (await getClient()).multi()
 
-  suffixes.map(suffix =>
-    ids.map(async id => {
-      await multi.hgetallAsync(decorateKey(`${prefix}:${id}:${suffix}`))
+  await suffixes.map(suffix =>
+    ids.map(id => {
+      return multi.hgetallAsync(decorateKey(`${prefix}:${id}:${suffix}`))
```

Comment:
Something still feels incorrect here. The inner map is going to return a `Promise[]` so then the outer map is going to return a `Promise[][]` which feels like an odd thing to `await`

Reply:
Also shouldn't this be a `Promise.all`?